src/main/kotlin/com/mineinabyss/launchy/logic/LaunchyState.kt

```diff
@@ -58,9 +67,18 @@ class LaunchyState(
     private var _deleted by mutableStateOf(0)
     val queuedDeletions by derivedStateOf {
         _deleted
-        disabledMods.filter { it.isDownloaded }.also { if(it.isEmpty()) updateNotPresent() }
+        disabledMods.filter { it.isDownloaded }.also { if (it.isEmpty()) updateNotPresent() }
     }
 
+    var notPresentConfigDownloads by mutableStateOf(setOf<Mod>())
```

Comment:
I'd prefer you evaluate this here directly instead of an `init` block, also the name is confusing. Is this supposed to be a list of mods with updates available?

Reply:
i just copied what we do with mods since they both basically should work identically